batchLogReader.py

```diff
@@ -275,8 +274,8 @@ def LogReader(filename,tier,movesets,ratings):
 			#	os.makedirs(d)
 			#msfile=open(outname,'ab')
 			if keyify(poke['species']) == 'meloettapirouette':
-				print filename
-			writeme={'trainer':trainer.encode('ascii', 'ignore'),
+				print(filename)
+			writeme={'trainer':trainer,
```

Comment:
Explain? This means the output could potentially be in unicode; it's unclear if that breaks anything later